python/GafferOSLUI/OSLObjectUI.py

```diff
@@ -70,7 +71,23 @@
 			"noduleLayout:section", "left",
 
 		],
+		"interpolation" : [
+
+			"description",
+			"""
+			Interpolation mode in which to process the object. 
+			All primitive variables are resampled to match the selected interpolation. 
```

Comment:
All non-constant primitive variables. Maybe we could explain a bit better what we mean by "in which to process the object"  as well? How about something like this?

"""
The interpolation type of the primitive variables created by this node. For instance, Uniform interpolation means that the shader is run once per face on a mesh, allowing it to output primitive variables with a value per face. All non-constant input primitive variables are resampled to match the selected interpolation so that they can be accessed from the shader.
"""